page-components/Review/ReviewSection/ViewReviewSection.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Container } from 'components/Container';
+import { GeneralFeedback, TimestampFeedback } from '.';
+import { Section } from 'components/Layout';
+import { mockGeneralFeedback } from 'mocks/api/Feedback';
+
+const CreateReviewSection = () => {
+  return (
+    <Container size='lg'>
+      <Section>
+        <TimestampFeedback mode='view' />
```

Comment:
Should this be view mode in the Create component?

Reply:
yea think its the view ocmponent, i just forgot to update comopnent name lol